### PR TITLE
Address the Review Comments of the SLP2 Release Note

### PR DESCRIPTION
**Pull request by praneesha**
## Purpose
Address the review comments of the slp2 release note.
> Fixes #

## Check List

- [ ] **Page Addition**
  - [ ] Add `permalink` to pages
  - [ ] If contains empty folder(s), Add front-matter `redirect_to:`

- [ ] **Page Rename**
  - [ ] Add front-matter `redirect_from`
  - [ ] Add front-matter `redirect_to:` (If applicable)

### CODE REVIEW

File: downloads/swan-lake-release-notes/swan-lake-preview2/RELEASE_NOTE.md [hunk 1]
```diff
@@ -243,26 +243,29 @@ The `config:getAsArray` and `config:getAsMap` functions now return immutable val
...
-- The `runtime:getProperty` function has been removed. Java interoperability can be used as the alternative to access Java system properties.
+**The new method:**
 
+- The `runtime:getProperty` function has been removed. Java interoperability can be used as the alternative to access Java system properties.
 - The auth-related configurations have been moved out from the `runtime:InvocationContext` to the newly-introduced `auth:InvocationContext`.
 
+```ballerina
+future<()> f1 = start runtime:sleep(50);
+```
+
```
Comment (pubudu91) [inline downloads/swan-lake-release-notes/swan-lake-preview2/RELEASE_NOTE.md:268]: This code block should come under the "The new method" heading. And I think it'd be better to leave "The new method" and "The previous method" without bold. Otherwise it might look like a heading, with the formatting in the website

File: downloads/swan-lake-release-notes/swan-lake-preview2/RELEASE_NOTE.md [hunk 2]
```diff
@@ -243,26 +243,29 @@ The `config:getAsArray` and `config:getAsMap` functions now return immutable val
...
-For example, when the following, which previously used the `runtime:timeout` is considered,
+**The previous method:**
+
+Used the `runtime:timeout` as shown below.
 
 ```ballerina
 future<()> f1 = runtime:timeout(50);
 ```
-can be done as follows:
-```ballerina
```
Comment (MaryamZi) [inline downloads/swan-lake-release-notes/swan-lake-preview2/RELEASE_NOTE.md:258]: This change has removed one example, and now "The new method" points to the next "The `runtime:getProperty` function has been removed. Java interoperability can be used as the alternative to access Java system properties.".

Please fix.

And like I mentioned on the original PR, we can't call this a new method.

File: downloads/swan-lake-release-notes/swan-lake-preview2/RELEASE_NOTE.md (hunk 1 above)
Comment (praneesha) [reply downloads/swan-lake-release-notes/swan-lake-preview2/RELEASE_NOTE.md:268]: Done.

File: downloads/swan-lake-release-notes/swan-lake-preview2/RELEASE_NOTE.md (hunk 2 above)
Comment (praneesha) [reply downloads/swan-lake-release-notes/swan-lake-preview2/RELEASE_NOTE.md:258]: Fixed.

File: downloads/swan-lake-release-notes/swan-lake-preview2/RELEASE_NOTE.md [hunk 3]
```diff
@@ -243,24 +243,25 @@ The `config:getAsArray` and `config:getAsMap` functions now return immutable val
...
-The WebSocket client now supports cookies. A new field `cookies` has been introduced in the `http:WebSocketClientConfiguration` and `http:WebSocketFailoverClientConfiguration` to specify the cookies.
+The WebSocket client now supports cookies. A new field `cookies` has been introduced in the `http:WebSocketClientConfiguration` and `http:WebSocketFailoverClientConfiguration` records to specify the cookies.
 
 ### Runtime API
 
-- The `runtime:timeout` function has been removed. The `runtime:sleep` function can be used as the alternative.
+The `runtime:timeout` function has been removed. The `runtime:sleep` function can be used as the alternative.
 
-For example, when the following, which previously used the `runtime:timeout` is considered,
+E.g., consider the following example, which uses the `runtime:timeout`.
```
Comment (MaryamZi) [inline downloads/swan-lake-release-notes/swan-lake-preview2/RELEASE_NOTE.md:252]:
```suggestion
E.g., consider the following example, which uses the `runtime:timeout` function.
```

File: downloads/swan-lake-release-notes/swan-lake-preview2/RELEASE_NOTE.md [hunk 4]
```diff
@@ -243,24 +243,25 @@ The `config:getAsArray` and `config:getAsMap` functions now return immutable val
...
 
 ```ballerina
-future<()> f1 = runtime:timeout(50);
+future<()> f1 = start runtime:sleep(50);
```
Comment (MaryamZi) [inline downloads/swan-lake-release-notes/swan-lake-preview2/RELEASE_NOTE.md:255]: Please undo this change.

File: downloads/swan-lake-release-notes/swan-lake-preview2/RELEASE_NOTE.md [hunk 5]
```diff
@@ -243,24 +243,25 @@ The `config:getAsArray` and `config:getAsMap` functions now return immutable val
...
 ```
-can be done as follows:
+
+This can now be done as follows:
+
 ```ballerina
-future<()> f1 = start runtime:sleep(50);
+future<()> f1 = runtime:timeout(50);
```
Comment (MaryamZi) [inline downloads/swan-lake-release-notes/swan-lake-preview2/RELEASE_NOTE.md:261]: This too.

File: downloads/swan-lake-release-notes/swan-lake-preview2/RELEASE_NOTE.md (hunk 4 above)
Comment (praneesha) [reply downloads/swan-lake-release-notes/swan-lake-preview2/RELEASE_NOTE.md]: Fixed.

File: downloads/swan-lake-release-notes/swan-lake-preview2/RELEASE_NOTE.md (hunk 5 above)
Comment (praneesha) [reply downloads/swan-lake-release-notes/swan-lake-preview2/RELEASE_NOTE.md]: Fixed.

File: downloads/swan-lake-release-notes/swan-lake-preview2/RELEASE_NOTE.md [hunk 6]
```diff
@@ -243,24 +243,25 @@ The `config:getAsArray` and `config:getAsMap` functions now return immutable val
 
 ### WebSocket
 
-The WebSocket client now supports cookies. A new field `cookies` has been introduced in the `http:WebSocketClientConfiguration` and `http:WebSocketFailoverClientConfiguration` to specify the cookies.
+The WebSocket client now supports cookies. A new field `cookies` has been introduced in the `http:WebSocketClientConfiguration` and `http:WebSocketFailoverClientConfiguration` records to specify the cookies.
 
 ### Runtime API
 
-- The `runtime:timeout` function has been removed. The `runtime:sleep` function can be used as the alternative.
+The `runtime:timeout` function has been removed. The `runtime:sleep` function can be used as the alternative.
```
Comment (MaryamZi) [inline downloads/swan-lake-release-notes/swan-lake-preview2/RELEASE_NOTE.md:250]: This needs to be a bullet point.
```suggestion
- The `runtime:timeout` function has been removed. The `runtime:sleep` function can be used as the alternative.
```

File: downloads/swan-lake-release-notes/swan-lake-preview2/RELEASE_NOTE.md (hunk 3 above)
Comment (praneesha) [reply downloads/swan-lake-release-notes/swan-lake-preview2/RELEASE_NOTE.md:252]: Fixed.

File: downloads/swan-lake-release-notes/swan-lake-preview2/RELEASE_NOTE.md (hunk 6 above)
Comment (praneesha) [reply downloads/swan-lake-release-notes/swan-lake-preview2/RELEASE_NOTE.md:250]: Fixed.

File: downloads/swan-lake-release-notes/swan-lake-preview2/RELEASE_NOTE.md (hunk 3 above)
Comment (MaryamZi) [reply downloads/swan-lake-release-notes/swan-lake-preview2/RELEASE_NOTE.md:252]: Can't seem to see this change.

Comment (praneesha) [reply downloads/swan-lake-release-notes/swan-lake-preview2/RELEASE_NOTE.md]: Its there.
